core/forking/gasSchedule.go

```diff
@@ -51,13 +64,13 @@ func NewGasScheduleNotifier(
 
 // RegisterNotifyHandler will register the provided handler to be called whenever a new epoch has changed
 func (g *gasScheduleNotifier) RegisterNotifyHandler(handler core.GasScheduleSubscribeHandler) {
-	if check.IfNil(handler) {
+	if check.IfNilReflect(handler) {
```

Comment:
why this change?

Reply:
it does not have isInterfaceNil. as that method is not pushed to vmcommon.

Reply:
ok